src/circuit/mod.rs

```diff
@@ -1,5 +1,10 @@
 #![allow(dead_code)]
 
+extern crate reqwest;
+
+use std::io;
+use std::fs::File;
```

Comment:
Only used in the tests, so you can call these imports when you start the test `mod`, after 
`#[cfg(test)]
mod test {`